apps/system/js/app_usage_metrics.js

```diff
@@ -90,6 +90,8 @@
   ];
 
 
+  const MARKETPLACE_ORIGIN = 'https://marketplace.firefox.com';
```

Comment:
A lot of other important things like this are configurable through the settings db. Do you want to make this configurable?

What about partners who want to have their own marketplace and want metrics? They'll be tempted to just switch this variable to point to their marketplace, and then we won't get data.  So maybe make this into an array of URLs so that partners can add others without removing ours?